targets/TARGET_NORDIC/ReadMe.md

```diff
@@ -0,0 +1,55 @@
+# Nordic platform's readme
```

Comment:
ok, for windows no difference since it is blind...
I will change to (the most popular in mbed-os) README.md